index.html

```diff
@@ -1,8 +1,60 @@
 <!DOCTYPE html>
 <html>
    <head>
+      <title>GA 4 - Ellen Brady - Blaze</title>
+      <link rel = "stylesheet" type = "text/css" href = "styles.css" />
    </head>
    <body>
-      <!-- Write your HTML in this file! -->
+      <header>
+        <p id="mainHeading">Pandemic Pup</p>
+      </header>
+         <main>
+           <article>
+            <section>
+                <a name="Blaze"></a>
+                <h1>Meet Blaze</h1>
+                <figure>
+                  <img id = "blaze" src="images/blaze.jpeg"  /> 
+                </figure>  
+                <div id="testP" class="funParagraph">Hello! Before we get into a few dog training tips, I would like to tell you a little bit about me. My name is Blaze and I am about 4 years old. Only I know of the places I've seen and what I have experienced prior to coming to the shelter in 2020. Yep, that's right - I'm what they call a pandemic pup. This can be a little challenging for my new family but they have been getting to know me more and more each day. It has been two years now that we've been together. Each day is full of exercise, affection and lots of training! My new mom has taught me so much! And now I'm here to share what I've learned in hopes to help other pups like myself.</p>
```

Comment:
When it comes to paragraphs like this, for readability, I recommend hitting enter after every sentence or so, so you don't have such a long single line where you need to scroll over so much. Don't worry, this won't change how it appears on your webpage, but just makes the code a little more readable!